tools/scripts/templates/template_launch.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Up until now, we stored these template files in the `tools/scripts/platform/{maxim, pico}` directory. I suggest we do the same for stm32

Reply:
As discussed the template files will be moved to tools/scripts/platform/ since the vs code support should be extended to other platforms and therefore would not fit well into a single platform folder.